### PR TITLE
remove stats piece

### PR DESCRIPTION
**Pull request by mikehelmick**
## Proposed Changes

* remove reference to daily request counts, we removed that code

**Release Note**

```release-note
NONE
```

### CODE REVIEW

**Review by whaught — COMMENTED** [review]
Screen Shot 2021-02-12 at 5 00 25 PM

The new TOC looks funny - is that right?

File: docs/api.md [hunk 1]
```diff
@@ -1,24 +1,4 @@
...
-    - [Handling batch partial success/failure](#handling-batch-partial-successfailure)
-  - [`/api/checkcodestatus`](#apicheckcodestatus)
-  - [`/api/expirecode`](#apiexpirecode)
-  - [`/api/stats/*`](#apistats)
-- [Chaffing requests](#chaffing-requests)
-- [Response codes overview](#response-codes-overview)
-
-<!-- /TOC -->
+<!-- TOC depthFrom:1 -->autoauto- [API access](#api-access)auto- [API usage](#api-usage)auto    - [Authenticating](#authenticating)auto    - [Error reporting](#error-reporting)auto- [API Methods](#api-methods)auto    - [`/api/verify`](#apiverify)auto    - [`/api/certificate`](#apicertificate)auto- [Admin APIs](#admin-apis)auto    - [`/api/issue`](#apiissue)auto        - [Client provided UUID to prevent duplicate SMS](#client-provided-uuid-to-prevent-duplicate-sms)auto    - [`/api/batch-issue`](#apibatch-issue)auto        - [Handling batch partial success/failure](#handling-batch-partial-successfailure)auto    - [`/api/checkcodestatus`](#apicheckcodestatus)auto    - [`/api/expirecode`](#apiexpirecode)auto    - [`/api/stats/*`](#apistats)auto- [Chaffing requests](#chaffing-requests)auto- [Response codes overview](#response-codes-overview)autoauto<!-- /TOC -->
```
Comment (sethvargo) [inline docs/api.md:1]: This is broken - can you install the vscode extension that does this automatically?

**Review by sethvargo — CHANGES_REQUESTED** [review]

File: docs/api.md (hunk 1 above)
Comment (mikehelmick) [reply docs/api.md]: fixed - don't know what happened there.
I have the TOC extension.

**Review by whaught — APPROVED** [review]

**Comment by mikehelmick** [comment]
TOC is fixed

**Pull request merged**